### ISSUE TITLE
Problems with Instances

### ISSUE THREAD

**Issue by striker873** [TanninOne/modorganizer#578]
Mod Organizer setects saves in /profile/Default/saves but game does not recongnize them... and saves them to /Documents/My Games/Fallout 4/Saves, any idea?

Aditionally I would like to know if there is any way to move the full /User/AppData/Mod Organizer/ folder to main Mod Organizer folder? Let's say: D:/Mod Organizer/Skyrim/profiles/Default/ ... estructure type...

**Comment by MyCoinsYourPurse** [comment]
If you choose the first option when first installing MO (the one that doesn't support multiple games), then the AppData Folder won't be touched.  

The problem is, this option never shows up again for me on re-installations.  Ever.

**Comment by striker873** [comment]
Yes, that would be a solution: having one installation of MO for each game. But this solution makes completly useless the multi instance feature of MO... 

Another problem I found: MO won't read the categories.dat fiile of Skyrim (in /user/AppData/Skyrim)... turns always on Fallout 4 categories (on installation I made the first game Fallout and them set Skyrim, checked all paths and they are ok)

**Comment by TanninOne** [comment]
Please create one report per issue, I can't track them properly otherwise.
